gclda/model.py

```diff
@@ -384,7 +384,7 @@ def _update_regions(self):
         subregions).
         """
         # Generate default ROI based on default_width
-        A = self.roi_size * np.eye(self.n_peak_dims)
+        default_roi = self.roi_size * np.eye(self.n_peak_dims)
```

Comment:
@adamkimbler Looks good. On a related note, do you have a solid handle on how the ROI definition works? I changed `roi` to `roi_size`, but I'm not sure what the units are. Is `roi_size` the diameter of an initial sphere, or is it the radius? Also, is it in mm or in voxels?